libs/application/templates/public-debt-payment-plan/src/fields/Overview/Overview.tsx

```diff
@@ -1,78 +1,147 @@
-              <Label>Nafn</Label>
-              <Text>Fjármundur Skuldason</Text>
-            </Box>
-            <Box>
-              <Label marginTop={2}>Sími</Label>
-              <Text>8486525</Text>
-            </Box>
+            {nationalRegistry?.fullName && (
+              <Box>
+                <Label>{formatMessage(overview.name)}</Label>
+                <Text>{nationalRegistry?.fullName}</Text>
+              </Box>
+            )}
+            {applicant?.phoneNumber && (
+              <Box>
+                <Label marginTop={2}>
+                  {formatMessage(overview.phoneNumber)}
+                </Label>
+                <Text>{applicant?.phoneNumber}</Text>
+              </Box>
+            )}
           </GridColumn>
           <GridColumn span={['6/12', '5/12']}>
-            <Box>
-              <Label>Heimilisfang</Label>
-              <Text>Skuldagötu 2, 110 Reykjavík</Text>
-            </Box>
-            <Box>
-              <Label marginTop={2}>Netfang</Label>
-              <Text>Fjarmundurskuldason@simnet.is</Text>
-            </Box>
+            {nationalRegistry?.address?.streetAddress &&
+              nationalRegistry?.address?.postalCode &&
+              nationalRegistry?.address?.city && (
+                <Box>
+                  <Label>{formatMessage(overview.address)}</Label>
+                  <Text>{`${nationalRegistry?.address?.streetAddress}, ${nationalRegistry?.address?.postalCode} ${nationalRegistry?.address?.city}`}</Text>
+                </Box>
+              )}
+            {applicant?.email && (
+              <Box>
+                <Label marginTop={2}>{formatMessage(overview.email)}</Label>
+                <Text>{applicant?.email}</Text>
+              </Box>
+            )}
           </GridColumn>
         </GridRow>
       </ReviewGroup>
-      <ReviewGroup isEditable editAction={editAction}>
+      <ReviewGroup
+        isEditable
+        editAction={() => editAction('employerMultiField')}
+      >
         <GridRow>
-          <GridColumn span={['6/12', '5/12']}>
-            <Box>
-              <Label>Vinnuveitandi</Label>
-              <Text>Krónan ehf.</Text>
-            </Box>
-          </GridColumn>
-          <GridColumn span={['6/12', '5/12']}>
-            <Box>
-              <Label>Kennitala vinnuveitanda</Label>
-              <Text>711298-2239</Text>
-            </Box>
-          </GridColumn>
+          {employer?.isCorrectInfo === YES && (
+            <GridColumn span={['6/12', '5/12']}>
+              <Box>
+                <Label>{formatMessage(overview.employer)}</Label>
+                <Text>{employerInfo.name}</Text>
+              </Box>
+            </GridColumn>
+          )}
+          {employerInfo?.nationalId && (
+            <GridColumn span={['6/12', '5/12']}>
+              <Box>
+                <Label>{formatMessage(overview.employerSsn)}</Label>
+                <Text>
+                  {employer?.isCorrectInfo === YES
```

Comment:
We have been exporting the answers to a constant file where they are defined like:
export const YES = 'yes'
export const NO = 'no'
And then importing that constant in the fields needed to have a defined constant instead of hardcoded string